app/helpers/admin/users_helper.rb

```diff
@@ -16,4 +16,9 @@ def change_admin_attribute(user)
       content_tag(:button)
     end
   end
+
+  def role(userrole)
```

Comment:
beware! helper uses global namespace. so the method role is defined everywhere in views. 
rename #role

Reply:
Done